src/main/resources/db/changelog/changes/11-create-cart-items-table.yaml

```diff
@@ -0,0 +1,40 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-cart-items-table
+      author: anton dudar
+      changes:
+        - createTable:
+            tableName: cart_items
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false

```

Comment:
You can delete nullable: false. It's redundant.

Reply:
agree, but it's up to you :) it's minor